spec/cart_spec.rb

```diff
@@ -205,7 +205,7 @@
 
     describe "#destroy" do
       it "should delete the line_item keys, the index key, and the cart key" do
-        cart.set_discounted 5
+        cart.set_discounted 'some code'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.